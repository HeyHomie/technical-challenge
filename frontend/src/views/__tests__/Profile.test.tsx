import { render } from '@testing-library/react'
import { Route, BrowserRouter as Router } from 'react-router-dom'
import { QueryParamProvider } from 'use-query-params'
import { ProfileView } from '..'

describe('Render Profile View', () => {
  it('should render Profile View', () => {
    const { container } = render(
      <Router>
        <QueryParamProvider ReactRouterRoute={Route}>
          <ProfileView />
        </QueryParamProvider>
      </Router>
    )
    expect(container.firstChild).toMatchSnapshot()
  })
})
