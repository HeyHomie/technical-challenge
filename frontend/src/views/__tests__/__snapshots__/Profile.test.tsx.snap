// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Profile View should render Profile View 1`] = `
<div>
  isLoading
</div>
`;
